tests/generator/test_make_fp.py

```diff
@@ -3,13 +3,15 @@
 import numpy as np
 import unittest
 import importlib
+import textwrap
```

Comment:
Why `textwrap` is imported?

Reply:
![image](https://user-images.githubusercontent.com/100290172/191443699-54c5970a-2270-470f-894a-4ebba0b73839.png)
I use it here

Reply:
This method is used to remove whitespaces but I don't see any whitespace in line 304-315.